embedded-graphics/src/primitives/circle/styled.rs

```diff
@@ -279,4 +298,49 @@ mod tests {
             );
         }
     }
+
+    #[test]
+    fn bounding_boxes() {
+        const CENTER: Point = Point::new(15, 15);
+        const SIZE: u32 = 10;
+
+        let style = PrimitiveStyle::with_stroke(BinaryColor::On, 3);
+
+        let center = Circle::with_center(CENTER, SIZE).into_styled(style);
+
+        let inside = Circle::with_center(CENTER, SIZE + 2).into_styled(
```

Comment:
If the test uses the `affected_area` function we don't need to calculate the size anymore.